packages/manager/modules/order/mock/order-details.ts

```diff
@@ -1,7 +1,6 @@
-import { PathParams } from 'msw';
-import { getParamsFromUrl } from '../../e2e/utils/playwright-helpers';
-import { Handler } from '../../e2e/utils/msw-helpers';
-import { OrderData, OrderDetail } from '@/api/order';
+import { Handler } from '../../../../../playwright-helpers';
```

Comment:
```suggestion
import { Handler } from '@playwright-helpers';
```

Reply:
the alias is not configured in the order package and it doesn't work well in playwright anyways